tools/ci_connector_ops/ci_connector_ops/sat_config_checks.py

```diff
@@ -0,0 +1,99 @@
+        )
+        if can_check_strictness_level:
+            try:
+                assert acceptance_test_config.get("test_strictness_level") == expected_test_strictness_level
+            except AssertionError:
+                connectors_with_bad_strictness_level.append(connector_name)
+    return connectors_with_bad_strictness_level
+
+def find_changed_ga_connectors() -> List[str]:
+    """Find GA connectors modified on the current branch.
+
+    Returns:
+        List[str]: The list of GA connector that were modified on the current branch.
+    """
+    changed_connector_names = utils.get_changed_connector_names()
+    return [connector_name for connector_name in changed_connector_names if utils.get_connector_release_stage(connector_name) == "generally_available"]
+
+def find_mandatory_reviewers() -> List[Union[str, Dict[str, List]]]:
+    teams = []
+    ga_connector_changes = find_changed_ga_connectors()
+    backward_compatibility_changes = utils.get_changed_acceptance_test_config(diff_regex="disable_for_version")
+    test_strictness_level_changes = utils.get_changed_acceptance_test_config(diff_regex="test_strictness_level")
+    if ga_connector_changes:
+        teams += GA_CONNECTOR_REVIEWERS
+    if backward_compatibility_changes:
+        teams += [{"any-of": list(BACKWARD_COMPATIBILITY_REVIEWERS)}]
+    if test_strictness_level_changes:
+        teams += [{"any-of": list(TEST_STRICTNESS_LEVEL_REVIEWERS)}]
+    return teams
+
+def check_test_strictness_level():
+    connectors_with_bad_strictness_level = find_connectors_with_bad_strictness_level()
+    if connectors_with_bad_strictness_level:
+        logging.error(
+            f"The following GA connectors must enable high test strictness level: {connectors_with_bad_strictness_level}. Please check this documentation for details: https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference/#strictness-level"
+        )
+        sys.exit(1)
+    else:
+        sys.exit(0)
+
+def write_review_requirements_file():
+    mandatory_reviewers = find_mandatory_reviewers()
+
+    if mandatory_reviewers:
+        requirements_file_content = [{
+            "name": "Required reviewers from the connector org teams",
+            "paths": "unmatched",
+            "teams": mandatory_reviewers
+        }]
+        with open(REVIEW_REQUIREMENTS_FILE_PATH, "w") as requirements_file:
+            yaml.safe_dump(requirements_file_content, requirements_file)
+        print(f"REVIEW_REQUIREMENTS_FILE={REVIEW_REQUIREMENTS_FILE_PATH}")
+    else:
+        print('REVIEW_REQUIREMENTS_FILE=""')
+
+def print_mandatory_reviewers():
+    teams = []
+    mandatory_reviewers = find_mandatory_reviewers()
+    for mandatory_reviewer in mandatory_reviewers:
+        if isinstance(mandatory_reviewer, dict):
+            teams += mandatory_reviewer["any-of"]
+        else:
+            teams.append(mandatory_reviewer)
```

Comment:
Doing the `any-of` check here feels a little un-DRY, since you also manually add `any-of` in `find_mandatory_reviewers` -- maybe you could put this logic of adding the any-of into a helper method you call in `find_mandatory_reviewers`?